Files/UserControls/NavigationToolbar.xaml

```diff
@@ -521,7 +522,7 @@
                     </Button.KeyboardAccelerators>
                 </Button>
 
-                <Grid x:Name="StatusCenterGrid" Visibility="{x:Bind ShowStatusCenter, Mode=OneWay}">
+                <Grid x:Name="StatusCenterGrid" Visibility="{x:Bind StatusCenterViewModel.AnyOperationsOngoing, Mode=OneWay}">
```

Comment:
This means that the user can't see if an operation completed successfully or not. This probably needs to be hidden if no operations were started OR if the user cleared all messages from the flyout

Reply:
That's a good point, do we have any properties that track if there are any uncleared messages in the flyout?

Reply:
What about checking if the status banner list count is 0?

Reply:
We would need to add a property for that.